src/lib.rs

```diff
@@ -572,10 +576,15 @@ impl<'a> GdbCommand<'a> {
     ///
     /// * `cmd` - gdb command parameter (-ex).
     pub fn ex(&mut self, cmd: &'a str) -> &'a mut GdbCommand {
```

Comment:
Use AsRef trait to allow both string and str:

https://doc.rust-lang.org/std/convert/trait.AsRef.html#

Reply:
Into?